test/integration/test_collections.js

```diff
@@ -66,6 +69,33 @@ module.exports = {
            expect(res.text).to.equal('test1,test2 collection(s) deleted');
            done();
          });
+    },
+
+    'test_collection_upload': function(done) {
```

Comment:
I like the test but you can remove the `ext` parameter by doing:
```javascript
    'test_collection_upload': function(done) {
      const test = exts => {
        if (!exts.length) {
          return done();
        }

        const ext = exts.shift();
        chai.request(SERVER_URL)
          .post(`${PATH_PREFIX}/collections/upload`)
          .attach('file', fs.readFileSync(`${__dirname}/fixture/import.${ext}`), `import.${ext}` )
          .set('Authorization', `Bearer ${TOKEN}`)
          .end((err, res) => {
            expect(err).to.be.null;
            expect(res).to.have.status(201);

            mongodbClient.dropCollection('import', test.bind(null, exts));
          });
      };

      test(['json','csv','bson']);
    }
```

Also can you fix the indentation on the function chain? 2 spaces like in my example above.